jest-puppeteer.config.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   server: {
-    command: `./node_modules/.bin/serve --listen ${process.env.JEST_PORT} --no-clipboard ./dist`,
+    command: `ruby -run -ehttpd ./dist -p${process.env.JEST_PORT}`,
```

Comment:
TIL! I had been doing `python -m SimpleHTTPServer` but it's nice to keep everything to one language

Reply:
> TIL! I had been doing `python -m SimpleHTTPServer` but it's nice to keep everything to one language

I actually used the Python version initially (07f1f5e) but:

- The command differs between Python 2 and Python 3
- Python isn't available in the CircleCI Ruby images, which makes sense in retrospect

Reply:
Also, of interest: https://gist.github.com/willurd/5720255